rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,157 @@
+---
+title: "Using RStudio Server on Windows"
+date: "Last updated on `r Sys.Date()`"
+output:
+  bookdown::html_document2:
+    keep_md: true
+    number_sections: false
+    toc: true
+toc-title: "Table of Contents"
+knit: "bookdown::render_book"
+---
+
+```{r klippy, echo=FALSE, include=TRUE}
+klippy::klippy(lang = c("bash", "r", "markdown"))
+
+knitr::opts_chunk$set(eval=FALSE, tidy = "styler")
+```
+
+## Introduction {#intro}
+
+This article explains about how to set up RStudio Server on Windows platform.
+
+As RStudio Server has primarily been designed to work with Linux distributions, Windows users must require a virtualization technology, called ["Windows Subsystem for Linux" (WSL)](https://docs.microsoft.com/en-us/windows/wsl/){target="_blank"}, to run a Linux kernel inside their system.
+
```

Comment:
Also add something like this at the end of the introduction section:

"We do not support running RStudio Server Pro on WSL at this time, and recommend RStudio Server on WSL only for single-user use, not as a way to deploy a multi-user server."